compiler/p/codegen/GenerateInstructions.hpp

```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2000, 2019 IBM Corp. and others
+ * Copyright (c) 2019, 2019 IBM Corp. and others
```

Comment:
The copyright date should not be changed.

Reply:
I believe the copyright is correct. Git is doing a poor job of showing it in the diff, but this is actually a new file. It just happens to have the same name as a file that's also renamed so git thinks the contents were moved. Looking at the diffs for each commit individually makes it easier to follow what actually happened.